lib/tasks/test/integration.rake

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+require "portus/cmd"
+require "portus/test"
+
+# TODO: on a Travis PR, only test portus:development and registry:2.{5,6}
+# TODO: otherwise, test with portus:2.3 and portus:head
```

Comment:
@vitoravelino this has been left out on purpose. I will tackle this in a subsequent PR. The idea is that running the whole matrix takes around 40 minutes on Travis, which is simply unbearable for mere PR. So, after merging this PR, I'll send another which:

- On PR it will just run the development image. In total it would then take around 20 minutes.
- If this is not a PR (e.g. a PR has just been merged and travis is performing a run against the fresh master branch), then we will test with 2.3 and head. This should take around 30 minutes.

If we see that overall we are taking too much time on Travis, then we could start thinking on parallelizing (e.g. running the integration tests on a Jenkins job instead of Travis).